src/main/java/graphql/execution/AsyncExecutionStrategy.java

```diff
@@ -36,11 +36,9 @@ public AsyncExecutionStrategy(DataFetcherExceptionHandler exceptionHandler) {
     public CompletableFuture<ExecutionResult> execute(ExecutionContext executionContext, ExecutionStrategyParameters parameters) throws NonNullableFieldWasNullException {
 
         Instrumentation instrumentation = executionContext.getInstrumentation();
-        InstrumentationExecutionStrategyParameters instrumentationParameters = new InstrumentationExecutionStrategyParameters(executionContext);
+        InstrumentationExecutionStrategyParameters instrumentationParameters = new InstrumentationExecutionStrategyParameters(executionContext, parameters);
 
-        InstrumentationContext<CompletableFuture<ExecutionResult>> executionStrategyCtx = instrumentation.beginExecutionStrategy(instrumentationParameters);
```

Comment:
I clean up and merged `beginExecutionStrategy` and `beginFields` into one step.  There are the same thing.

The beginFields was a mistake in retrospect.